android/app/src/main/res/layout/activity_main.xml

```diff
@@ -36,18 +36,17 @@
                 app:layout_scrollFlags="scroll|enterAlways"
                 app:popupTheme="@style/PopupOverlay" />
 
-        </android.support.design.widget.AppBarLayout>
-
-
-        <ProgressBar
-            android:id="@+id/progress"
-            style="@style/Base.Widget.AppCompat.ProgressBar.Horizontal"
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="?attr/actionBarSize"
-            android:max="100"
-            android:visibility="gone" />
+            <ProgressBar
+                android:id="@+id/progress"
+                android:layout_marginBottom="-6dp"
```

Comment:
Please move this to dimens instead, rather than hardcoding the values :slightly_smiling_face: